lib/commons/dom/get-parsed-resource.js

```diff
@@ -0,0 +1,100 @@
+/* global dom */
+
+/**
+ * Parse resource object for a given node from a specified attribute
+ * @method getParsedResource
+ * @param {HTMLElement} node given node
+ * @param {String} attribute attribute of the node from which resource should be parsed
+ * @returns {Object}
+ */
+dom.getParsedResource = function getParsedResource(node, attribute) {
```

Comment:
This name doesn't really describe that it does. How about `urlPropsFromAttribute`?